### ISSUE TITLE
Openshot tracks users when starting

### ISSUE THREAD

**Issue by cometchaserde** [OpenShot/openshot-qt#3282]
Today I noticed that while starting Openshot from within a terminal it loads a webpage from Google.
Looks like tracking the users
e.g.
http://www.google-analytics.com/collect?ua=Mozilla%2F5.0+%28X11%3B+Linux+x86_64%29+AppleWebKit%2F537.36+%28KHTML%2C+like+Gecko%29+Chrome%2F37.0.2062.120+Safari%2F537.36&tid=UA-4381101-5&cd5=debian-10.3-&cd=close-app&v=1&cd2=3.4.3&aid=org.openshot.openshot-qt&cd3=5.2.1&cd4=5.2.1&cd1=0.2.4&aip=1&cid=b825ee66-f179-4ca4-9c71-1c39c378c3ca&sc=end&t=screenview&an=OpenShot+Video+Editor&av=2.5.0&ul=en-us

**Comment by ferdnyc** [comment]
@cometchaserde 

OpenShot can collect fully anonymized statistics from user sessions, yes.

(Stats about the OS environment and the installed versions of things like Python, Qt, and PyQt5 are also logged, but no personal information is included simply because there _isn't any_ to send — OpenShot doesn't ask for or collect any identifying information for its users. Activity stats and crash reports get associated with a randomly-generated identifier that's stored in the user's settings file.)

The initial tutorial window that's shown on first launch provides the ability to opt out of data collection, and that choice can be revisited at any time in the Debug tap of the Preferences dialog:

![image](https://user-images.githubusercontent.com/538020/76155377-533dd980-60b9-11ea-8f89-8e3a67933e4b.png)

In the past OpenShot _would_ send some initial telemetry about the start of the session, **before** the user had the chance to opt out — but that was changed in 2.5.0. Now, initial startup metrics data is collected and queued until after the tutorial starts and the opt-out choice is displayed.

Long story short, if you'd prefer anonymous data not be shared, go into Preferences > Debug and uncheck "Send Anonymous Metrics and Errors" to turn off the metrics collection.

**Comment by ferdnyc** [comment]
Duplicate of #1129 #877 #776 #123

**Comment by bnavigator** [comment]
FTR, this changes the opt-out to opt-in:

```diff
diff -Nur openshot-qt-2.6.1/src/settings/_default.settings new/src/settings/_default.settings
--- openshot-qt-2.6.1/src/settings/_default.settings	2021-09-05 01:16:20.000000000 +0200
+++ new/src/settings/_default.settings	2022-07-10 11:36:30.694748033 +0200
@@ -340,7 +340,7 @@
     "restart": true
   },
   {
-    "value": true,
+    "value": false,
     "title": "Send Anonymous Metrics and Errors",
     "type": "bool",
     "category": "Debug",
```

**Comment by bnavigator** [comment]
#4735
